AxonDeepSeg/segment.py

```diff
@@ -24,23 +26,52 @@
 from config import axonmyelin_suffix, axon_suffix, myelin_suffix
 
 # Global variables
-SEM_DEFAULT_MODEL_NAME = "default_SEM_model"
+SEM_DEFAULT_MODEL_NAME = "model_seg_rat_axon-myelin_sem"
 TEM_DEFAULT_MODEL_NAME = "default_TEM_model"
-OM_MODEL_NAME = "model_seg_pns_bf"
```

Comment:
Done too

Reply:
There is an error in CLI with this one:
TypeError: __init__() got an unexpected keyword argument 'filter_empty_mask_patch'

This is because the 'filter_empty_mask_patch' key doesn't exist in the current version of ivadomed (and is not even yet on ivadomed master). I did not think about that before publishing the model.
For segmentation we could remove it and it would work (it is only relevant in training).
I'll think about that and get back to you tomorrow.

Reply:
Hi @mathieuboudreau!
I updated the [mb/v4 branch](https://github.com/axondeepseg/default-OM-model/tree/mb/v4) with a fix for the model.

For reference, the fix is:
- I removed the `filter_empty_mask_patch` key from the config json file used for segmentation by ivadomed (model_seg_rat_axon-myelin_bf.json).
- I added a config file specifically for training that includes the key (model_seg_rat_axon-myelin_bf_training.json) and updated the README accordingly.

Reply:
Great, thanks! I've released a new version and updated this branch for that.